### DISCUSSION TITLE
Hole placement on ancestral pueblan style flute.

### DISCUSSION THREAD

**Discussion by stanleyloftness** [edwardkort/WWIDesigner#151] [Flute Study]
I am having trouble understanding how to use flute study.  I want to determine hole placement for a ancestral Puebla style flute.  .875" inside bore, .125" wall thickness, 31.7" long, 2mm indent, in Key of G.  Can anyone help?

**Comment by bhp1** [comment]
For a rim-blown flute, you are dealing with an in-line air stream and no head space, rather than a side-blown air stream with possible head space above the embouchure hole.  You might be better off using the whistle study model rather than the flute study model; at least it offers a more compatible definition of the mouthpiece position.  Unfortunately, with a rim-blown flute, pretty much all of the other mouthpiece geometry parameters - window length, window width, window height, and windway height - are ill-defined and under the control of the player.

Before you start drilling holes, it would be a good idea to test how well WIDesigner predicts the tuning of a rim-blown flute, and also get some idea how to set the mouthpiece geometry parameters.  Do you have a finished flute at hand, even just a prototype?  You could measure the geometry and tuning, and see how well WIDesigner's predictions match the measurements.

Suggestions for setting the mouthpiece parameters:

window length - Estimate how far the player's lips are from the rim (splitting edge).
window width - Estimate the area of the open space between lips and rim, and divide by the window length.
windway height - Estimate the height of the space between the player's lips.
windway length - (This value isn't used right now, so it doesn't matter what you set it to.)
window height and beta factor - Use the whistle calibrator to set these.
